### DISCUSSION TITLE
Is it possible to use wakapi in obsidian?

### DISCUSSION THREAD

**Discussion by StepaniaH** [muety/wakapi#723] [Q&A] [answered]
Hey experts!

I installed the wakatime plugin of obsidian, and I wonder that is it possible to config the url and key like neovim to track my usage in obsidian?

**Comment by muety** [comment, marked as answer]
You should be able to configure API key and -url by editing the `.wakatime.cfg` file as described [here](https://github.com/muety/wakapi?tab=readme-ov-file#-client-setup). Looking at the plugin's code, there also appears to be a second `.wakatime-internal.cfg` file, so if things don't work, perhaps try having a look at that file as well.

If things still won't work, please have a look at WakaTime's log file (`~/.wakatime/wakatime.log`) and Obsidian's [logs](https://forum.obsidian.md/t/where-can-i-find-logfiles/55434).

**Reply by StepaniaH** [reply]
Thanks, it works. I do not know that it use the same .cfg file before, my bad, thanks again.
